cmd/lakectl/cmd/abuse.go

```diff
@@ -344,7 +343,7 @@ func init() {
 	abuseCmd.AddCommand(abuseCreateBranchesCmd)
 	abuseCreateBranchesCmd.Flags().String("branch-prefix", "abuse-", "prefix to create branches under")
 	abuseCreateBranchesCmd.Flags().Bool("clean-only", false, "only clean up past runs")
-	abuseCreateBranchesCmd.Flags().Int("amount", 1000000, "amount of things to do")
+	abuseCreateBranchesCmd.Flags().Int("amount", 100000, "amount of things to do")
```

Comment:
nice
I would use const
Do you think the default values for the other commands(read, write, commit, list) here are OK?

Reply:
1. Will change to `const`
2. Will revert to 1m - I wanted to see the end